### PR TITLE
Implemented edit and show function and css styles

### PR DESCRIPTION
**Pull request by AnaCarrizales**
"Mostrar" function shows the details of the event and it is possible to edit that specific event.

Screenshot from 2024-01-29 18-10-33
Screenshot from 2024-01-29 18-11-20
Screenshot from 2024-01-29 18-11-43

### CODE REVIEW

**Review by juperameza — APPROVED** [review]
lgtm

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -10,7 +10,11 @@
   </head>
   <body>
     <%= render 'home/header' %>
+    <div class="main-wrapper">
+    <div class="container">
     <%= yield %>
+    </div>
+    </div>
```
Comment (MauricioNC) [inline app/views/layouts/application.html.erb]: Indent correctly this code.

**Review by MauricioNC — CHANGES_REQUESTED** [review]

File: app/views/events/_form.html.erb [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<%= form_with(model: @evento, local: true, class:"form-wrapper") do |form| %>
+  <% if @evento.errors.any? %>
```
Comment (MauricioNC) [inline app/views/events/_form.html.erb]: It is not necessary to use "@ evento", you are already passing this variable when rendering the form "evento: @ evento". Just use it as "evento".

**Review by MauricioNC — CHANGES_REQUESTED** [review]

**Review by BrandonVG — APPROVED** [review]

**Review by MauricioNC — APPROVED** [review]
LGTM

**Pull request merged**